modules/data.land/DESCRIPTION

```diff
@@ -14,7 +14,10 @@ Description: The Predictive Ecosystem Carbon Analyzer (PEcAn) is a scientific
 Depends:
     PEcAn.DB,
     PEcAn.utils,
-    dbplyr
+    dbplyr, 
+    redland,
+    datapack,
+    dataone,
```

Comment:
Consider moving dataone from the `Depends` second to the `Imports` section just below, then include the namespace in the spelling each time you use a `dataone` function inside `data.land`. So for example if you're using the (hypothetical) dataone function `some_func`, you'd write `dataone::some_func(...)`.

Reply:
Sounds good. I already use that format in my dataone related functions. To confirm, I can move dataone to `Imports`? Should I do anything to its dependencies (`redland` and `datapack`)?

Reply:
Yes, you can move dataone to Imports. If redland and datapack are never called directly from any data.land functions, they should be able to go unmentioned in this file and dataone will handle its own dependency on them for itself.